### PR TITLE
Add Python 3.7 to CI.

### PR DESCRIPTION
**Pull request by garenchan**
Python 3.7.0 has been released at 2018-06-27. So I think we need to add it to CI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,10 +4,16 @@ language: python
 
 matrix:
   include:
-    - python: 3.6
+    - python: 3.7
+      dist: xenial
+      sudo: true
```
[the remark this answers is not in the document]
Comment (garenchan) [reply .travis.yml]: Yes. If we don't add this, the CI service will fail to download 3.7 archive.

```
3.7 is not installed; attempting download
Downloading archive: https://s3.amazonaws.com/travis-python-archives/binaries/ubuntu/14.04/x86_64/python-3.7.tar.bz2
$ curl -sSf -o python-3.7.tar.bz2 ${archive_url}
curl: (22) The requested URL returned error: 403 Forbidden
Unable to download 3.7 archive. The archive may not exist. Please consider a different version.
```

[the remark this answers is not in the document]
Comment (ThiefMaster) [reply .travis.yml]: I meant only the sudo line, not the dist

[the remark this answers is not in the document]
Comment (garenchan) [reply .travis.yml]: My fault, this sudo line is not needed.

[the remark this answers is not in the document]
Comment (davidism) [reply .travis.yml]: PRs to our other projects have been setting `dist: xenial` globally. `sudo` is not needed.

[the remark this answers is not in the document]
Comment (garenchan) [reply .travis.yml]: OK, I'm going to put the `dist: xenial` setting in the global scope.

**Pull request merged**